src/main/java/com/isa/morswiny/eventsDao/EventRepository.java

```diff
@@ -37,4 +39,55 @@ public Event getEventByID(Integer id) {
         }
         return null;
     }
+
+    @Override
+    public boolean isEventExisting(Event event) {
+        return Objects.nonNull(event) && allEventsList.contains(event);
+    }
+
```

Comment:
ja bym to zmienil, bo jak event == null to wladujemy do listy. Na cos takiego:
if (Objects.nonNull(event)) {
      return allEventsList.contains(event);
} 
return true; //zeby nie wladowac pustego eventu